src/vs/workbench/browser/labels.ts

```diff
@@ -514,7 +514,7 @@ class ResourceLabelWidget extends IconLabel {
 		const resource = toResource(this.label);
 		const label = this.label.name;
 
-		if (this.options && typeof this.options.title === 'string') {
+		if (this.options && (typeof this.options.title === 'string' || typeof this.options.title === 'object')) {
```

Comment:
```suggestion
		if (this.options && (typeof this.options.title === 'string' || this.options.title instanceof Promise)) {
```

Reply:
I ended up simplifying to `this.options.title !== undefined`